Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/worker/DataDonationAnalyticsWorkBuilder.kt

```diff
@@ -25,5 +27,11 @@ class DataDonationAnalyticsWorkBuilder @Inject constructor() {
                 BackgroundConstants.BACKOFF_INITIAL_DELAY,
                 TimeUnit.MINUTES
             )
+            .setConstraints(buildConstraints())
```

Comment:
Would it make sense to add a log if the worker wasn't started due to the network constraint?

Reply:
I'm not sure if we can, the work is scheduled by the WorkManager api and if there is no network connection the worker is simply not spawned, i don't think we get any kind of callback

Reply:
we could maybe remove the constraint and do the check inside of the worker ourselves but i think this kind of defeats the purpose of the work manager in the first place